libraries/botbuilder-dialogs/src/memory/dialogStateManager.ts

```diff
@@ -33,7 +34,7 @@ export interface DialogStateManagerConfiguration {
  * of turn state. Path resolvers allow for shortcut behavior for mapping things like 
  * $foo -> dialog.foo
  */
-export class DialogStateManager {
+export class DialogStateManager implements MemoryInterface {
```

Comment:
You shouldn't actually need to do this given that TypeScript uses Duck Typing.  This would prevent you from having to add a dependency on botframework-expressions,

Reply:
> You shouldn't actually need to do this given that TypeScript uses Duck Typing. This would prevent you from having to add a dependency on botframework-expressions,

Okay, I'll remove this.